py-polars/tests/unit/series/test_series.py

```diff
@@ -2328,20 +2328,6 @@ def test_extend_constant(const: Any, dtype: pl.PolarsDataType) -> None:
     assert_series_equal(s.extend_constant(const, 3), expected)
 
 
-def test_any_all() -> None:
```

Comment:
Is this intended?

Reply:
Yes, above that in the same test file is `test_all_any()` which is parametrized and appears to do a lot more.

Reply:
Sorry I misspoke, it's not in the same file, but there is a [`test_all_any.py`](https://github.com/pola-rs/polars/blob/main/py-polars/tests/unit/series/test_all_any.py) that has much more extensive parametrized tests so this isn't needed here.